soc/arm/nordic_nrf/nrf53/Kconfig.soc

```diff
@@ -129,6 +129,16 @@ config SOC_DCDC_NRF53X_HV
 	help
 	  Enable nRF53 series System on Chip High Voltage DC/DC converter.
 
+config SOC_ENABLE_LFXO
+	bool "Enable LFXO"
```

Comment:
it should be enabled only if LFXO is used.
maybe add
```
default n if CLOCK_CONTROL_NRF_K32SRC_SYNTH
default n if CLOCK_CONTROL_NRF_K32SRC_RC
```

Reply:
Is that actually desirable? It won't hurt to configure the pins for LFXO even if it is not used. And with the above added, it would not be possible to configure APP to use RC (e.g. temporarily, before starting NET) and NET to use LFXO.